mslice/plotting/plot_window/cut_plot.py

```diff
@@ -193,14 +194,28 @@ def toggle_errorbar(self, line_index, line_options):
                 element.set_alpha(1)
 
     def is_icut(self, is_icut):
+
+        if is_icut:
+            self.plot_window.action_keep.triggered.connect(self.keep_triggered)
+            self.plot_window.action_make_current.triggered.connect(self.make_current_triggered)
+
```

Comment:
This works but I think there's a simpler way. `is_icut` is always called when interactive cuts is turned on or off, which is the important behaviour here. So I think we can just do:
```
 self._canvas.manager.button_pressed_connected(not is_icut)
 self._canvas.manager.picking_connected(not is_icut)
```
Also, I agree that this method is poorly named. I think it can be changed to something like 'set_interactive_cut_mode`. 
```

Reply:
Alright